src/plugins/data-card/data-card-tile.tsx

```diff
@@ -27,7 +30,8 @@ export const DataCardToolComponent: React.FC<ITileProps> = observer((props) => {
   const [imageUrlToAdd, setImageUrlToAdd] = useState<string>("");
   const shouldShowAddCase = !readOnly && isTileSelected;
   const shouldShowDeleteCase = !readOnly && isTileSelected && content.dataSet.cases.length > 1;
-  const shouldShowAddField = !readOnly && isTileSelected;
+  const displayAs = content.selectedSortAttributeName === "none" ? "single" : "sorted";
```

Comment:
I'd change this to `displaySingle`, a boolean. It would be true if there isn't a specified `selectedSortAttributeName` (or `Id`), plus you could validate that the name/id references an actual attribute (in case the name changed, or the selected attribute was deleted, it would make sense to display a single instead).

Reply:
That's kind of what I'm going to push shortly, but I didn't think of the case of an invalid menu item....will post back.

Reply:
it's a boolean now